test/airbrake/utils_test.exs

```diff
@@ -48,4 +46,76 @@ defmodule Airbrake.UtilsTest do
              }
     end
   end
+
+  describe "detuple/1" do
+    test "returns the original data when input is a struct with no tuples" do
+      input = %Struct{baz: 100, qux: 200}
```

Comment:
I'm thinking we could test the recursion here (too).

```suggestion
      input = %Struct{baz: {1, 2}, qux: {:ok, "foobar"}}
```

Similarly for rest of the tests, make sure that nested tuples are detupled.

Reply:
2433bad5cfd03dab5d264f3a2c3a3d2f07986de2